staging/src/k8s.io/client-go/kubernetes/typed/core/v1/pod_expansion.go

```diff
@@ -30,7 +31,9 @@ import (
 // The PodExpansion interface allows manually adding extra methods to the PodInterface.
 type PodExpansion interface {
 	Bind(ctx context.Context, binding *v1.Binding, opts metav1.CreateOptions) error
-	Evict(ctx context.Context, eviction *policy.Eviction) error
+	Evict(ctx context.Context, eviction *policyv1beta1.Eviction) error
```

Comment:
I feel like there should be a comment explaining what will happen? Do you get beta or v1? Or does it try one and fall back?

Reply:
added a comment and marked deprecated